### PR TITLE
feat: PC-12978 Add composite examples

### PR DESCRIPTION
**Pull request by BSski**
## Motivation

Repo was lacking composite SLO example.

## Summary

Added missing example.

### CODE REVIEW

File: examples/resources/nobl9_slo/resource.tf [hunk 1]
```diff
@@ -123,3 +123,46 @@ resource "nobl9_slo" "this" {
...
+    display_name = "OK"
+    name         = "tf-objective-1"
+    target       = 0.8
+    value        = 1
+    composite {
+      max_delay = "45m"
+      components {
+        objectives {
+          composite_objective {
+            project      = nobl9_slo.slo_1.project.name
```
Comment (annatrifonova) [inline examples/resources/nobl9_slo/resource.tf:151]: Right now error appears on apply:
```
 Error: Unsupported attribute
│ 
│   on main.tf line 170, in resource "nobl9_slo" "composite_slo":
│  170:             project      = nobl9_slo.slo_1.project.name
│ 
│ Can't access attributes on a primitive-typed value (string).
```
```suggestion
            project      = nobl9_slo.slo_1.project
```

File: examples/resources/nobl9_slo/resource.tf [hunk 2]
```diff
@@ -123,3 +123,46 @@ resource "nobl9_slo" "this" {
...
+            slo          = nobl9_slo.slo_1.name
+            objective    = "tf-objective-1"
+            weight       = 0.8
+            when_delayed = "CountAsGood"
+          }
+          composite_objective {
+            project      = nobl9_slo.slo_2.project.name
```
Comment (annatrifonova) [inline examples/resources/nobl9_slo/resource.tf:158]:
```suggestion
            project      = nobl9_slo.slo_2.project
```

**Review by annatrifonova — CHANGES_REQUESTED** [review]

File: examples/resources/nobl9_slo/resource.tf (hunk 1 above)
Comment (BSski) [reply examples/resources/nobl9_slo/resource.tf]: Good catch, I modified it after applying, thanks! :smile:

**Review by annatrifonova — APPROVED** [review]
lgtm :)

**Pull request merged**